src/shared/components/QueryBuilder/domain/helpers/__tests__/queryToFilters.spec.ts

```diff
@@ -79,6 +79,28 @@ const cases: TestCase[] = [
       },
     ],
   ],
+  [
+    ' process_cpu:cpu:nanoseconds:cpu:nanoseconds{service_name="core-requests",free_text=~"one,two"}',
+    [
+      {
+        id: expect.any(String),
+        type: FilterKind['attribute-operator-value'],
+        active: true,
+        attribute: {
+          label: 'free_text',
+          value: 'free_text',
+        },
+        operator: {
+          label: '=~',
+          value: '=~',
+        },
+        value: {
+          label: 'one,two',
+          value: 'one,two',
+        },
+      },
+    ],
+  ],
   // TODO: uncomment when we'll support the "in" operator
```

Comment:
Can this be removed now?

Reply:
Yes, but only after finalizing https://github.com/grafana/explore-profiles/pull/122

Reply:
Yep, sorry I was looking at two PRs at the same time and got confused :D